psyneulink/core/compositions/composition.py

```diff
@@ -10867,6 +10867,23 @@ def _delete_contexts(self, *contexts, check_simulation_storage=False, visited=No
             except AttributeError:
                 self.scheduler._delete_counts(c)
 
+    def _initialize_as_agent_rep(self, context, base_context, alt_controller=None):
```

Comment:
should this assert that only one of `self.controller` and `alt_controller` is `not None`?

Reply:
Actually I'll have to check on what's intended here. If that's true, I'll add the assert